src/main/java/seedu/address/logic/commands/MailEventCommand.java

```diff
@@ -0,0 +1,92 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_EVENT_DISPLAYED_INDEX;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.opencsv.CSVWriter;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.event.Event;
+import seedu.address.model.event.UidList;
+import seedu.address.model.person.Person;
+
+/**
+ * Creates mailing list of customer name and email address in an event.
+ */
+public class MailEventCommand extends Command {
+    public static final String COMMAND_WORD = "mailEvent";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Creates mailing list of all persons in an event in csv format, "
+            + "and stores the csv file in the /data folder. "
+            + "Parameters: "
+            + "EVENT INDEX (must be a positive integer)\n"
```

Comment:
Should we add the underscore for this?
```suggestion
            + "EVENT_INDEX (must be a positive integer)\n"
```